nearcore/src/config.rs

```diff
@@ -1225,6 +1227,7 @@ pub fn load_config(
         let signer = InMemoryValidatorSigner::from_file(&validator_file).with_context(|| {
             format!("Failed initializing validator signer from {}", validator_file.display())
         })?;
+        assert!(!config.tracked_shards.is_empty(), "Validator must track all shards. Please change `tracked_shards` field in config.json to be any non-empty vector");
```

Comment:
You can also add link to: https://github.com/near/nearcore/issues/7388 in the message